app/controllers/migration_controller.rb

```diff
@@ -38,18 +35,19 @@ def errors
       @aggregation.edm_isShownBy.errors.full_messages
   end
 
-  def new_aggregation
+  def new_aggregation(attributes = {})
     ORE::Aggregation.new(aggregation_defaults).tap do |aggregation|
-      aggregation.edm_aggregatedCHO.build_dc_contributor
-      aggregation.edm_aggregatedCHO.build_dc_creator
-      aggregation.build_edm_isShownBy
-
-      autocomplete(aggregation.edm_aggregatedCHO, :dc_subject, url: vocabularies_unesco_path, param: 'q')
-      autocomplete(aggregation.edm_aggregatedCHO.dc_creator, :rdaGr2_placeOfBirth, url: vocabularies_europeana_places_path, param: 'q')
-      autocomplete(aggregation.edm_aggregatedCHO.dc_creator, :rdaGr2_placeOfDeath, url: vocabularies_europeana_places_path, param: 'q')
+      aggregation.update(attributes)
```

Comment:
this actually persists the attributes to the db if everything is valid, is this intentional in the flow?
That's what was causing the form to send patch instead of post requests on the reCAPTCHA branch.

Perhaps it would be enough to only set each of the attributes from the params, instead of calling update?

Reply:
Quite right, this was quite wrong, addressed in 3b21a599ba6ea3f0d43e160a7636bc3f8ebc48d2